src/clr-angular/forms/styles/_variables.forms.scss

```diff
@@ -79,3 +79,12 @@ $clr-forms-radio-top: $clr-forms-checkbox-top;
 $clr-forms-radio-selected-shadow: $clr-forms-checkbox-checked-shadow;
 $clr-forms-radio-focused-shadow: 0 0 $clr-outline-blur $clr-outline-spread $clr-outline-color;
 $clr-forms-radio-disabled-shadow: $clr-forms-checkbox-checked-shadow;
+
+// Range
+$clr-forms-range-progress-fill-color: $clr-color-action-600 !default;
+$clr-forms-range-track-color: $clr-color-neutral-200 !default;
+$clr-forms-range-track-height: 4px; //$clr_baselineRem_4px; //4px
+$clr-forms-range-thumb-height: 14px; //$clr_baselineRem_0_583; //14px
+$clr-forms-range-thumb-margin: -5px; //-1 * $clr_baselineRem_5px; //-5px
+$clr-forms-range-margin: 5px; //$clr_baselineRem_5px; //5px
```

Comment:
I understand why thumb and even track height display strangely with rem values. But I don't see how margins would. Please fix.